src/views/Conference.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <Page class="league-page-container">
+    <div
+      class="conference-page"
+    >
+      <div
+        id="jitsi"
+      />
+    </div>
+  </Page>
+</template>
+<script>
+// eslint-disable-next-line import/no-extraneous-dependencies
+import JitsiMeetExternalAPI from 'jitsi-iframe-api';
+import Page from '../components/layout/Page.vue';
+import { IS_MOBILE_DEVICE } from '../utils/util';
+
+export default {
+  name: 'Conference',
+  components: {
+    Page,
+  },
+  mounted() {
+    // eslint-disable-next-line no-new
+    new JitsiMeetExternalAPI('test.league.aeternity.org', {
```

Comment:
Have you implemented the actual JWT authentication inside this package?

Reply:
look at the 
https://github.com/aeternity/jitsi-meet/blob/9d6505e60e6e7a0fa68b4d359773ca4874bdb8f8/react/features/conference/components/web/Conference.js#L228

Reply:
it's implemented at jitsi client side.